streaming/index.js

```diff
@@ -273,7 +275,9 @@ const startServer = async () => {
 
     if (subs[channel].length === 0) {
       logger.debug(`Subscribe ${channel}`);
-      redisSubscribeClient.subscribe(channel, (err, count) => {
+
+      // FIXME: https://github.com/redis/ioredis/issues/1910
+      redisSubscribeClient.subscribe(`${redisNamespace}${channel}`, (err, count) => {
```

Comment:
This is probably the better way of handling the keyPrefix instead of littering every `subscribe` and `unsubscribe` callsite with the prefix.